core/src/main/scala/com/salesforce/op/evaluators/EvaluationMetrics.scala

```diff
@@ -181,6 +179,8 @@ object RegressionEvalMetrics extends Enum[RegressionEvalMetric] {
   case object MeanSquaredError extends RegressionEvalMetric("mse", "mean square error", false)
   case object R2 extends RegressionEvalMetric("r2", "r2", true)
   case object MeanAbsoluteError extends RegressionEvalMetric("mae", "mean absolute error", false)
+  case object SignedPercentageErrorHistogram extends RegressionEvalMetric("signedpercentageerrorhistogram",
```

Comment:
"signedpercentageerrorhistogram" is not readable - how about "speh" or "prcntErrorHst"